mws/resource_mws_workspaces.go

```diff
@@ -340,10 +340,20 @@ func (a WorkspacesAPI) List(mwsAcctID string) ([]Workspace, error) {
 }
 
 type Token struct {
-	LifetimeSeconds int32  `json:"lifetime_seconds,omitempty" tf:"default:2592000"`
-	Comment         string `json:"comment,omitempty" tf:"default:Terraform PAT"`
-	TokenID         string `json:"token_id,omitempty" tf:"computed"`
-	TokenValue      string `json:"token_value,omitempty" tf:"computed,sensitive"`
+	LifetimeSeconds int32           `json:"lifetime_seconds,omitempty" tf:"default:2592000"`
+	Comment         string          `json:"comment,omitempty" tf:"default:Terraform PAT"`
+	TokenID         string          `json:"token_id,omitempty" tf:"computed"`
+	TokenValue      SensitiveString `json:"token_value,omitempty" tf:"computed,sensitive"`
+}
+
+type SensitiveString string
```

Comment:
nit: we can probably move this to Go SDK and also have DebugString method call these methods